web-crypto/sign-verify/index.html

```diff
@@ -83,20 +146,48 @@ <h2 class="sign-verify-heading">HMAC</h2>
           <section class="sign-verify-controls">
             <div class="message-control">
               <label for="hmac-message">Enter a message to sign:</label>
-              <input type="text" id="hmac-message" name="message" size="25"
-                     value="The bunny hops at teatime">
+              <input
+                type="text"
+                id="hmac-message"
+                name="message"
+                size="25"
+                value="The bunny hops at teatime"
+              />
+            </div>
+            <div class="signature">
+              Signature:<span class="signature-value"></span>
+            </div>
+            <input class="sign-button" type="button" value="Sign" />
+            <input class="verify-button" type="button" value="Verify" />
+          </section>
+        </section>
+
+        <section class="sign-verify ed25519">
+          <h2 class="sign-verify-heading">ED25519</h2>
```

Comment:
Here is one more nitpick that I missed before:
```suggestion
          <h2 class="sign-verify-heading">Ed25519</h2>
```
Other than that, it looks good to me :+1:

Reply:
Thank you, merging shortly 👍🏻